server/express/package.json

```diff
@@ -13,10 +14,14 @@
     "body-parser": "^1.19.0",
     "cookie-parser": "^1.4.5",
     "express": "^4.17.1",
+    "fs": "^0.0.1-security",
```

Comment:
I don't think this should be added as it's included in node https://www.npmjs.com/package/fs